src/course_bot/presentation.clj

```diff
@@ -602,6 +604,10 @@
             (talk/send-text token id (tr :pres/already-received))
             (talk/stop-talk tx))
 
+          (when (> (misc/read-time dt) now)
+            (talk/send-text token id (tr :pres/too-early))
```

Comment:
Do you mean" "You can't five a feedback to the future lesson"?

Also, we need a test for it.